docs/design/design.md

```diff
@@ -63,17 +63,17 @@ For more see [node-image.md][node-image.md].
 
 ### Cluster Creation
 
-Each "node" runs as a docker container. Each container initially boots to a
+Each "node" runs as a container. Each container initially boots to a
 pseudo "paused" state, with [the entrypoint][entrypoint] waiting for `SIGUSR1`.
-This allows us to manipulate and inspect the container with `docker exec ...`
+This allows us to manipulate and inspect the container with `containerEngine exec ...`
```

Comment:
I think this sort of change is going to be more confusing than anything. podman is the only tool I know of besides docker that provides something like this, and does so by mimicing the docker CLI AFAICT... "docker exec" is searchable when looking for upstream doc / what it is "containerEngine exec" is not.

how about "with eg `docker exec`" - it seems like podman users will still understand what this maps to.

Reply:
> how about "with eg docker exec" - 
or something like:
> with tools like `docker exec`